src/components/LazyLog/index.tsx

```diff
@@ -734,7 +749,10 @@ export default class LazyLog extends Component<any, any> {
                 formatPart={this.handleFormatPart(number)}
                 selectable={selectableLines}
                 highlight={highlight.includes(number)}
-                onLineNumberClick={this.handleHighlight}
+                onLineNumberClick={(e) => {
+                    this.handleHighlight(e);
+                    onLineNumberClick(number);
```

Comment:
Shouldn't this first check if `onLineNumberClick` is null or not? I think with typescript you can do `onLineNumberClick?.(number)`